lib/util/ops.py

```diff
@@ -201,6 +208,29 @@ def lrelu(x, leak=0.2, name="lrelu"):
         f2 = 0.5 * (1 - leak)
         return f1 * x + f2 * abs(x)
 
+# http://stackoverflow.com/questions/39975676/how-to-implement-prelu-activation-in-tensorflow
+prelu_count = 0
+def prelu(prefix):
+    def prelu_internal(_x):
+        global prelu_count
+        prelu_count += 1
+        name = (prefix+"prelu_"+str(prelu_count))
+        orig_shape = _x.get_shape()
+        _x = tf.reshape(_x, [24, -1])
```

Comment:
We hardcode batch size here - not good.  There's set_ops_type above which is a global variable hack for eventually supporting float16.